instrumentation/net/http/otelhttp/internal/semconvutil/httpconv.go

```diff
@@ -286,6 +294,38 @@ func (c *httpConv) ClientRequest(req *http.Request) []attribute.KeyValue {
 	return attrs
 }
 
+// ClientRequestMetrics returns metric attributes for an HTTP request made by a client. The
+// following attributes are always returned: "http.method", "net.peer.name".
+// The following attributes are returned if the related values
+// are defined in req: "net.peer.port".
+func (c *httpConv) ClientRequestMetrics(req *http.Request) []attribute.KeyValue {
```

Comment:
Maybe it could be replaced with
semconv.HTTPClientAttributesFromHTTPRequest(req) ?

Reply:
The `semconv` does not have such function.

Reply:
yes, it was deprecated at version 12. It was replaced with httpconv.ClientRequest(req) from "go.opentelemetry.io/otel/semconv/v1.20.0/httpconv"